### PR TITLE
Improvement femr 126 add descriptions for each setting

### PR DESCRIPTION
**Pull request by judyvbaxter**
Descriptions populate for each system setting and display on the Configure page. 

https://teamfemr.atlassian.net/browse/FEMR-126

### CODE REVIEW

File: app/femr/ui/models/admin/configure/IndexViewModelGet.java [hunk 1]
```diff
@@ -4,21 +4,33 @@
 import java.util.Map;
 
 public class IndexViewModelGet {
-    private Map<String, Boolean> settings;
+    private Map<String, Boolean> settings;;
+    private Map<String, String> descriptions;;
```
Comment (kevinzurek) [inline app/femr/ui/models/admin/configure/IndexViewModelGet.java]: @judyvbaxter can you comment what the Maps contain so someone new can understand why we have 2 maps here? :+1:

Comment (judyvbaxter) [inline app/femr/ui/models/admin/configure/IndexViewModelGet.java]: Done, pushed up the changes. Added a comment that clarifies that the first map is a name-isActive pair, and the second is a name-description pair.

File: app/femr/ui/models/admin/configure/IndexViewModelPost.java [hunk 2]
```diff
@@ -4,12 +4,21 @@
...
+        return descriptions;
+    }
+
     public void setSettings(List<String> settings) {
         this.settings = settings;
     }
+
+    public void setDescriptions(List<String> settings) {
+        this.descriptions = descriptions;
+    }
```
Comment (kevinzurek) [inline app/femr/ui/models/admin/configure/IndexViewModelPost.java]: Do we need these methods in the POST view model?

File: app/femr/ui/models/admin/configure/IndexViewModelPost.java [hunk 3]
```diff
@@ -4,12 +4,14 @@
...
 public class IndexViewModelPost {
     private List<String> settings;
+    private List<String> descriptions;
 
     public List<String> getSettings() {
         return settings;
     }
 
-    public void setSettings(List<String> settings) {
-        this.settings = settings;
```
Comment (kevinzurek) [inline app/femr/ui/models/admin/configure/IndexViewModelPost.java]: @judyvbaxter removing this causes an error when you submit the settings. This is supposed to catch the settings during a POST

Comment (judyvbaxter) [inline app/femr/ui/models/admin/configure/IndexViewModelPost.java]: I've added this method back in.

**Comment by kevinzurek** [comment]
I don't think this is needed either in the POST view model:

https://github.com/FEMR/femr/pull/562/files#diff-e45bd89c919bfb9100b1e46517054458R13

the POST view model should only contain variables for data you are trying to capture from the web page when a user clicks submit and sends the data back to the server. The descriptions aren't necessary to retrieve (they are just there for the user to see).

**Comment by kevinzurek** [comment]
Fixed formatting, rebased, and merged manually

**Pull request closed without merging**